### ISSUE TITLE
Imported module is stored in shadowed identifier with concatenated modules on

### ISSUE THREAD

**Issue by joeywatts** [webpack/webpack#10308]
# Bug report

**What is the current behavior?**

I'm using webpack to bundle a `web` target that depends on [material-ui](https://github.com/mui-org/material-ui/). With the `concatenateModules` optimization setting on, it incorrectly chooses a shadowed identifier to store module imports.

In `ButtonBase.js`, there are two similar identifiers - `_focusVisible` and `focusVisible`:
```javascript
  var _focusVisible = require("../utils/focusVisible");
  // ...
  var _React$useState = _react.default.useState(false),
      focusVisible = _React$useState[0],
      setFocusVisible = _React$useState[1];

  if (disabled && focusVisible) {
    setFocusVisible(false);
  }

  var _useIsFocusVisible = (0, _focusVisible.useIsFocusVisible)(),
      isFocusVisible = _useIsFocusVisible.isFocusVisible,
      onBlurVisible = _useIsFocusVisible.onBlurVisible,
      focusVisibleRef = _useIsFocusVisible.ref;
```

When this module is concatenated with the `TouchRipple.js` and `Ripple.js` modules which live in the same directory (turning minimize option off so we can analyze the output), I see this in the output:

```javascript
// EXTERNAL MODULE: ./node_modules/@material-ui/core/esm/utils/focusVisible.js
var focusVisible = __webpack_require__(8310);
// ...
  var _React$useState = react_default().useState(false),
      focusVisible = _React$useState[0],
      setFocusVisible = _React$useState[1];

  if (disabled && focusVisible) {
    setFocusVisible(false);
  }

  var _useIsFocusVisible = (0,focusVisible/* useIsFocusVisible */.a)(),
      isFocusVisible = _useIsFocusVisible.isFocusVisible,
      onBlurVisible = _useIsFocusVisible.onBlurVisible,
      focusVisibleRef = _useIsFocusVisible.ref;
```

It's using `focusVisible` as the variable to store the module, but that identifier is shadowed in the place where the module is actually used.

When I disable module concatenation, it uses the `_utils_focusVisible__WEBPACK_IMPORTED_MODULE_6__` identifier to store the module, so the conflict does not occur.

**What is the expected behavior?**

The identifier chosen for the `./node_modules/@material-ui/core/esm/utils/focusVisible.js` external module should not be shadowed at the usage site (ie, the two `focusVisible` variables in the above example should be assigned unique names).

**Other relevant information:**
webpack version: webpack@5.0.0-beta.12
Node.js version: v12.14.1
Operating System: macOS 10.14.5
Additional tools: N/A

**Comment by sokra** [comment]
should be fixed in 5.0.0-beta.13

**Comment by joeywatts** [comment]
Thanks @sokra! That was fast!
